src/corelibs/U2View/src/ov_sequence/DetViewSequenceEditor.h

```diff
@@ -73,7 +73,10 @@ class DetViewSequenceEditor : public QObject {
 
 private slots:
     void sl_editMode(bool active);
-    void sl_changeCursorColor();
+
+    /** Called periodically as a timer callback to animate blinking edit-mode cursor. */
```

Comment:
If we use [C++ documentation style](https://developer.lsst.io/cpp/api-docs.html) we will be able to generate code documentation one day (just in case, who knows, maybe it will be useful)

Reply:
I think that IDE (MSVC, QtCreator, CLion) are much better code browsers than HTML browser. 

In the documentation style I use I usually avoid mandatory tags and trivial statements and only put the additional information to the one that is already clear from the C++ method/field declaration.